src/pages/atoms/index.tsx

```diff
@@ -1,33 +1,173 @@
 import Button from "../../components/atoms/buttons/Button";
 import { Card } from "../../components/atoms/frame/Card";
+import TextInput from "../../components/atoms/inputs/TextInput";
+import PasswordInput from "../../components/atoms/inputs/PasswordInput";
+import { type FieldError } from "react-hook-form";
+import TextArea from "../../components/atoms/inputs/TextArea";
+import EmailInput from "../../components/atoms/inputs/TextInput";
+import { useForm } from "react-hook-form";
+import { z } from "zod";
+import { zodResolver } from "@hookform/resolvers/zod";
+
+const mockError: FieldError = {
+  type: "required",
+  message: "This field is required",
+};
+
+const mockPasswordError: FieldError = {
+  type: "required",
+  message: "Password incorrect",
+};
```

Comment:
use this instead 

```suggestion
const mockPasswordError: FieldError = {
  type: "required",
  message: "Password incorrect",
};
```
ok